examples/ffi/crashtracking.c

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+#include <datadog/common.h>
+#include <datadog/profiling.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <signal.h>
+
+void example_segfault_handler(int signal) {
+    printf("Segmentation fault caught. Signal number: %d\n", signal);
+    exit(-1);
+}
+
+int main(int argc, char **argv) {
+  if (signal(SIGSEGV, example_segfault_handler) == SIG_ERR) {
+    perror("Error setting up signal handler");
+    return -1;
+  }
+
+  ddog_prof_CrashtrackerConfiguration config = {
+    .create_alt_stack = false,
+    .endpoint = ddog_Endpoint_agent(DDOG_CHARSLICE_C("http://localhost:8126")),
```

Comment:
⛏️ for local testing purposes, might be better to route this to a file.  Adding some machinery to make this easier https://github.com/DataDog/libdatadog/pull/343